index.html

```diff
@@ -7,11 +7,10 @@
 <div class="bg_full">
   <div class="container container_narrow">
 
-    <h3><span class="label label-info">Important</span></h3>
-    <h2>Hackership Batch-2 on a Beach!</h2>
-    <p>Hackership, the full-time coding master class run by the <a href="/berlin/">OpenTechSchool Berlin</a>, openend its application for Batch-2 taking place Winter 2014/2015 in Gran Canaria. Learn more about it, the schedule and deadline through the <a href="http://blog.hackership.org/2014/10/Destination%20Announcement.html" target="_blank">announcement blog post</a> or at one of their <a href="http://blog.hackership.org/2014/10/Taste%20of%20Hackership%20is%20Back.html" target="_blank">Taste of Hackership sessions</a>.
+    <h2>OpenTechSchool Org.</h2>
+    <p>Hi, we are OpenTechSchool, a distributed community initiative organising <b>welcoming and inclusive code learning</b> happenings with real people in the real world. Our offers are for technology enthusiasts of all ages, genders, backgrounds, and experience levels. On top, we invite anyone to become an organiser themselves, set up their own <a href="/locator/">local chapter</a> organising learning events and <a href="http://discourse.opentechschool.org/">join our community</a> of coaches everywhere in creating original teaching material under the open <a href="http://creativecommons.org/licenses/by-sa/3.0/deed.en_US" target="_blank">Creative Commons Licence</a>.
```

Comment:
Ugh, didn't we have a good description somewhere?  Some stuff in there sounds really involved, like *“real people in the real world”* (as if finding somebody to meet up with in the tech world would be hard or uncommon), or *“code learning happenings”* (why are we no longer doing tech workshops?).